docs/DICTIONARIES.md

```diff
@@ -10,10 +10,17 @@ This project uses `dictionaries` to unify props between different components.
 ## List of dictionaries
 
 - [Size](#size)
+- [Color](#color)
 
 ### Size
 
 | Dictionary    | Values                               | Code name    |
 | ------------- | ------------------------------------ | ------------ |
 | Size          | `small`, `medium`, `large`           | Size         |
 | Size Extended | Size Dictionary + `xsmall`, `xlarge` | SizeExtended |
+
+### Color
+
+| Dictionary    | Values                                        | Code name    |
+| ------------- | --------------------------------------------- | ------------ |
+| Emotion Color | `success`, `danger`, `informative`, `warning` | EmotionColor |
```

Comment:
Can we sort the color base on the level they express -> success, informative, warning, danger, I mean...

Reply:
Sorted everywhere (I hope).